Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION = 0.5.2
+VERSION = 0.6.0
```

Comment:
What is the makefile still used for ? If it is still used, can I read information from the package.json so that we only need to change version numbers in **one** place